src/ng/directive/ngController.js

```diff
@@ -147,20 +147,20 @@
  * <example name="ngController" module="controllerExample">
  *  <file name="index.html">
  *   <div id="ctrl-exmpl" ng-controller="SettingsController2">
- *     Name: <input type="text" ng-model="name"/>
- *     [ <a href="" ng-click="greet()">greet</a> ]<br/>
+ *     <label>Name: <input type="text" ng-model="name"/></label>
+ *     <button ng-click="greet()">greet</button><br/>
  *     Contact:
  *     <ul>
  *       <li ng-repeat="contact in contacts">
- *         <select ng-model="contact.type">
+ *         <select ng-model="contact.type" id="select_{{$index}}">
```

Comment:
Cool, good call with the `aria-labelledby`! I'll defer to @gkalpak on this approach to interpolating the `$index` value, I'm not sure if there are other approaches to getting the index of the chosen option in the `select`.